applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/prj.conf

```diff
@@ -85,6 +85,9 @@ CONFIG_BT_CTLR_DATA_LENGTH_MAX=35
 
 CONFIG_ENTROPY_CC3XX=n
 
+# Don't use BT TX processor thread due to limited RAM.
```

Comment:
Please keep it right under `CONFIG_BT_LONG_WQ=n` to keep configurations consistent (also in other configurations)